pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -201,10 +212,25 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) reconcile(request reconcile.Requ
 	// DELETE: the CR was deleted
 	if !cephFilesystemSubVolumeGroup.GetDeletionTimestamp().IsZero() {
 		logger.Debugf("deleting subvolume group %q", namespacedName)
+
+		cephFsSvgList := &cephv1.CephFilesystemSubVolumeGroupList{}
+		namespaceListOpts := client.InNamespace(cephCluster.Namespace)
+		// List cephFilesystemSubvolumeGroup CR based on filesystem and spec.name
+		matchingKey := fmt.Sprintf("%s/%s", cephFilesystemSubVolumeGroup.Spec.FilesystemName, getSubvolumeGroupName(cephFilesystemSubVolumeGroup))
+		err = r.client.List(r.opManagerContext, cephFsSvgList, &client.MatchingFields{cephSVGFileSystemNameIndex: matchingKey}, namespaceListOpts)
+		if client.IgnoreNotFound(err) != nil {
+			return reconcile.Result{}, errors.Wrap(err, "failed to list cephFilesystemSubvolumeGroup")
+		}
+
 		// On external cluster, we don't delete the subvolume group, it has to be deleted manually
 		if cephCluster.Spec.External.Enable {
 			logger.Warningf("external subvolume group %q deletion is not supported, delete it manually", namespacedName)
-		} else {
+		} else if len(cephFsSvgList.Items) <= 1 {
```

Comment:
How about logging that we are allowing deletion when more than 1 svg is found?

Reply:
added